File: Java/wow_factor.java

/**
 * Write a description of wow_factor here.
 * 
 * @author (your name) 
 * @version (a version number or a date)
 */
import java.lang.String;
import java.util.Scanner;
public class wow_factor {
    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        String s=sc.nextLine();
        long ans=0,count=0,v=0;
        for(int i=0;i<s.length();i++){
            if(s.charAt(i)=='o') count+=v;
            else if(i>0&&s.charAt(i-1)=='v'){
                v++;
                ans+=count;
            }
        }
        System.out.println(ans);
    }
}

File: Python/wow_factor.py
import os
from math import*

s=input()
v,count,ans=[0]*3

for i in range(len(s)):
	if s[i]=="o":
		count+=v
	elif i>0 and s[i-1]=="v":
		v+=1
		ans+=count
print(ans)
